src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -99,13 +105,13 @@
   }
 
   > :first-child:not(.#{$text-input-group}__text) {
+    margin-block-start: var(--#{$text-input-group}__main--first-child--not--text-input--MarginBlockStart);
+    margin-block-end: var(--#{$text-input-group}__main--first-child--not--text-input--MarginBlockEnd);
```

Comment:
I think this runs the risk of causing the text input group component to potentially be too tall depending on what that first-child is. The height needs to match other controls/form elements - button, menu toggle, text input, etc. That height is 37px in v6 (it was 36px in v5), and the text input group with the label group is 38px tall in this PR. If you put this text input group in a filter group next to another control (like a button), they'll be different heights.

In v5, we added padding to the chip group main element when it's in text input group, I wonder if that might work here, too? https://www.patternfly.org/components/text-input-group/html#filters